README.md

```diff
@@ -1,137 +1,183 @@
+    transactionHash:
+      '0x82c13aaac6f0b6471afb94a3a64ae89d45baa3608ad397621dbb0d847f51196f',
+    decodeTxData: true,
+  });
+};
 ```
 
-#### Get account balances
+#### `getAccountBalance`
+
+Get the coin and token balances of a wallet.
+
 ```javascript
 const balances = async () => {
-    return await provider.getAccountBalance({
-        blockchain: "eth",
-        walletAddress: "0xfa9019df60d3c710d7d583b2d69e18d412257617"
-    })
-}
+  return await provider.getAccountBalance({
+    blockchain: 'eth',
+    walletAddress: '0xfa9019df60d3c710d7d583b2d69e18d412257617',
+  });
+};
 ```
 
-#### Get NFTs
+#### `getNFTsByOwner`
+
+Get data about all the NFTs (collectibles) owned by a wallet.
+
+````javascript
+
 ```javascript
 const nfts = async () => {
-    return await provider.getNFTsByOwner({
-        blockchain: "eth",
-        walletAddress: "0x0E11A192d574b342C51be9e306694C41547185DD",
-        filter: [
-            {"0x700b4b9f39bb1faf5d0d16a20488f2733550bff4": []},
-            {"0xd8682bfa6918b0174f287b888e765b9a1b4dc9c3": ["8937"]}
-        ]
-    })
-}
-```
-#### Get token holders
+  return await provider.getNFTsByOwner({
+    blockchain: 'eth',
+    walletAddress: '0x0E11A192d574b342C51be9e306694C41547185DD',
+    filter: [
+      { '0x700b4b9f39bb1faf5d0d16a20488f2733550bff4': [] },
+      { '0xd8682bfa6918b0174f287b888e765b9a1b4dc9c3': ['8937'] },
+    ],
+  });
+};
+````
+
+#### `getTokenHolders`
+
+Get the list of token holders for a given contract address.
+
 ```javascript
 const tokenHolders = async () => {
-    return await provider.getTokenHolders({
-        blockchain: "eth",
-        contractAddress: "0xdac17f958d2ee523a2206206994597c13d831ec7"
-    })
-}
+  return await provider.getTokenHolders({
+    blockchain: 'eth',
+    contractAddress: '0xdac17f958d2ee523a2206206994597c13d831ec7',
+  });
+};
 ```
-#### Get token holders count history
+
+#### `getTokenHoldersCount`
+
+Get current and historical data about the number of token holders for a given contract address.
+
 ```javascript
 const tokenHoldersCount = async () => {
-    return await provider.getTokenHoldersCount({
-        blockchain: "eth",
-        contractAddress: "0xdac17f958d2ee523a2206206994597c13d831ec7"
-    })
-}
+  return await provider.getTokenHoldersCount({
+    blockchain: 'eth',
+    contractAddress: '0xdac17f958d2ee523a2206206994597c13d831ec7',
+  });
+};
 ```
-#### Get currencies
+
+#### `getCurrencies`
+
+Get a list of supported currencies for a given blockchain.
+
 ```javascript
 const currencies = async () => {
-    return await provider.getCurrencies({blockchain: "fantom"})
-}
+  return await provider.getCurrencies({ blockchain: 'fantom' });
+};
 ```
+
+### About API keys
+
+For now, Ankr is offering _free_ access to these APIs with no request limits i.e. you don't need an API key at this time.
+
+Later on, these APIs will become a part of Ankr Protocol's [Premium Plan](https://www.ankr.com/protocol/plan/).
```

Comment:
Moved the API key section to the bottom and slightly re-worded it since it's not that important for devs trying this out right now